aci/data_source_aci_fvsubnet.go

```diff
@@ -61,7 +62,6 @@ func dataSourceAciSubnet() *schema.Resource {
 
 			"virtual": &schema.Schema{
 				Type:     schema.TypeString,
-				Optional: true,
```

Comment:
I see you have changed to computed for attributes in data source and in the documentation. Is there a reason to skip next_hop_addr, msnlb + nested attributes, and anycast_mac? These can also all be changed to computed from what I see.